CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ librdkafka v2.1.1 is a maintenance release:
    calling `rd_kafka_message_leader_epoch()` on the polled `rkmessage` (#4245).
  * Fix a segmentation fault when fetching from follower and the partition lease
    expires while waiting for the result of a list offsets operation (#4254).
+ * Fix documentation for request timeout, incorrectly stating -1 for infinte
```

Comment:
Let's say "for the admin request timeout". In the admin client the request timeout is the local one, while the operation timeout is sent to the broker. There is also the "request.timeout.ms" that applies to the produce request and is sent to the broker.
```suggestion
 * Fix documentation for the admin request timeout, incorrectly stating -1 for infinite
```